AutocompleteClient/FW/Logic/Request/CIOTrackQuizConversionData.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CIOTrackQuizConversionData.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+/**
+ Struct encapsulating the parameters that must/can be set set in order to track quiz conversion
+ */
+struct CIOTrackQuizConversionData: CIORequestData {
+
+    let quizID: String
+    let quizVersionID: String
+    let quizSessionID: String
+    let customerID: String
+    let variationID: String?
+    let itemName: String?
+    let revenue: Double?
+    let conversionType: String?
+    let isCustomType: Bool?
+    let displayName: String?
+    var sectionName: String?
+
+    func url(with baseURL: String) -> String {
+        return String(format: Constants.trackQuizConversion.format, baseURL)
+    }
+
+    init(quizID: String, quizVersionID: String, quizSessionID: String, customerID: String, variationID: String? = nil, itemName: String? = nil, revenue: Double? = nil, conversionType: String? = nil, isCustomType: Bool? = nil, displayName: String? = nil, sectionName: String? = nil) {
+        self.quizID = quizID
+        self.quizVersionID = quizVersionID
+        self.quizSessionID = quizSessionID
+        self.customerID = customerID
+        self.variationID = variationID
+        self.itemName = itemName
+        self.revenue = revenue
+        self.conversionType = conversionType
+        self.isCustomType = isCustomType
+        self.displayName = displayName
+        self.sectionName = sectionName
+    }
+
+    func decorateRequest(requestBuilder: RequestBuilder) {
+        requestBuilder.set(autocompleteSection: self.sectionName)
+    }
+
+    func httpMethod() -> String {
+        return "POST"
+    }
+
+    func httpBody(baseParams: [String: Any]) -> Data? {
+        var dict = [
+            "quiz_id": self.quizID,
+            "quiz_version_id": self.quizVersionID,
+            "quiz_session_id": self.quizSessionID,
+            "item_id": self.customerID,
+            "action_class": "conversion"
```

Comment:
Let's remove `action_class` here since it defaults to it on the backend. No reason for us to specify it here as well.